eslzArm/eslz-portal.json

```diff
@@ -2179,6 +2179,26 @@
                                 ]
                             }
                         },
+                        {
+                            "name": "denyPipOnNICforCorp",
```

Comment:
```suggestion
                            "name": "denyPipOnNicForCorp",
```

Reply:
why this here? I have the same in other places, so it would seem fair to change this everywhere(?)

Reply:
i think its just to enforce camelCasing, not a show stopper if used elsewhere. But you have mixed here between camel and non camel, so just make consistent and all good :)